space_packet_parser/xtce/definitions.py

```diff
@@ -40,82 +46,81 @@ class XtcePacketDefinition(common.AttrComparable):
     #  This will require an additional namespace dict entry for xsi (in this example)
     def __init__(
             self,
-            container_list: Optional[list[containers.SequenceContainer]] = None,
+            container_set: Optional[set[containers.SequenceContainer]] = None,
```

Comment:
I think we want to allow any iterable thing. Maybe indicate that it should be unique or we warn if it isn't.
```suggestion
            container_set: Optional[Iterable[containers.SequenceContainer]] = None,
```